.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,5 +11,11 @@
 * Enter your extended release note here
 ```
 
+If you want this PR to close an issue add the following:
+"fixes #<issue nubmber>"
```

Comment:
this should be `backtick quoted` or else the `<issue nubmber>` gets eaten as HTML

"nubmber" typo